SingularityUI/app/components/taskSearch/DisplayResults.cjsx

```diff
@@ -0,0 +1,242 @@
+                        </div>
+                        <button type="submit" className="btn btn-default">Jump!</button>
+                        &nbsp;&nbsp;
+                        <div className='form-group'>
+                            <label htmlFor='count'>Tasks Per Page: </label>
+                            &nbsp;&nbsp;
+                            <DropDown
+                                forceChooseValue = true
+                                value = @props.count
+                                choices = {@props.countChoices}
+                                inputType = 'number'
+                                id = 'count'
+                                title = 'Tasks Per Page'
+                                updateFn = @updateCount />
+                        </div>
+                        &nbsp;&nbsp;
+                        <div className='form-group'>
+                            <label htmlFor="sortDirection">Sort Direction:</label>
+                            &nbsp;&nbsp;
+                            <DropDown
+                                forceChooseValue = true
+                                value = @props.sortDirection
+                                choices = Enums.sortDirections()
+                                inputType = 'sortDirection'
+                                id = 'sortDirection'
+                                title = 'Sort Direction'
+                                updateFn = @props.updateSortDirection />
+                        </div>
+                    </form>
+                </ul>
+            </nav>
+        </div>
+
+
+    renderTasks: ->
+        taskTableColumns = ["Name", "Last State", "Deploy", "Started", "Updated"]
+        taskTableData = []
+        for task in @collection.models
+            taskTableData.push([<Link
+                                    text={task.taskId.id}
+                                    url={window.config.appRoot + "/task/" + task.taskId.id}
+                                    altText={task.taskId.id}
+                                />, 
+                                <TaskStateLabel
+                                    taskState={task.lastTaskState}
+                                />,
+                                <Link
+                                    text={task.taskId.deployId}
+                                    url={window.config.appRoot + "/request/" + task.taskId.requestId + "/deploy/" + task.taskId.deployId}
+                                />, 
+                                <TimeStamp 
+                                    timestamp={task.taskId.startedAt} 
+                                    display='timeStampFromNow'} 
+                                />, 
+                                <TimeStamp 
+                                    timestamp={task.updatedAt} 
+                                    display='timeStampFromNow'} 
+                                />])
+        return <StripedTable
```

Comment:
Probably should have a wrapper component that generates the `StripedTable` and all of the data in `taskTableData`